pkg/util/testing/wrappers.go

```diff
@@ -359,3 +374,29 @@ func (rf *ResourceFlavorWrapper) Taint(t corev1.Taint) *ResourceFlavorWrapper {
 	rf.Taints = append(rf.Taints, t)
 	return rf
 }
+
+// RuntimeClassWrapper wraps a RuntimeClass.
+type RuntimeClassWrapper struct{ nodev1.RuntimeClass }
+
+// MakeRuntimeClass creates a wrapper for a Runtime.
+func MakeRuntimeClass(name, handler string) *RuntimeClassWrapper {
```

Comment:
Is the handler required?

Reply:
Yes, an empty handler will be rejected in creation.